haxe.html.markdown

```diff
@@ -11,7 +12,7 @@ Haxe author).  Note that this guide is for Haxe version 3.  Some of the guide
 may be applicable to older versions, but it is recommended to use other
 references.
 
-```csharp
+```haxe
```

Comment:
Sorry, but haxe [is not supported](https://github.com/adambard/rouge/tree/master/lib/rouge/lexers) in the website. Will have to use csharp as it is the best alternative.